### PR TITLE
feat: add cart page

### PR DESCRIPTION
**Pull request by Vikas2426**
image

### CODE REVIEW

**Review by shobit1337 — APPROVED** [review]
Everything looks good to merge!

File: card/card.css [hunk 1]
```diff
@@ -37,3 +38,23 @@
...
   color: var(--black-light);
 }
+
+.card-horizontal {
+  display: flex;
+  width: 100%;
+  background-color: var(--white);
+  overflow: hidden;
+  padding: 0.5rem;
+  background-color: white;
```
Comment (manismk) [inline card/card.css:48]: remove one background color

**Review by manismk — APPROVED** [review]
Looks good

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JA-Store</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="flex align-center">
+        <a href="./index.html">
+            <span class="brand">MyWebsite</span>
```
Comment (shraddha15-creator) [inline cart.html:15]: I would suggest you give some good name to your e-commerce app.

**Review by shraddha15-creator — APPROVED** [review]

File: card/card.css (hunk 1 above)
Comment (Vikas2426) [reply card/card.css]: done

File: cart.html (hunk 2 above)
Comment (Vikas2426) [reply cart.html]: done

**Pull request merged**